site/plugins.json

```
@@ -422,5 +422,13 @@
     "package": "@netlify/plugin-nextjs",
     "repo": "https://github.com/netlify/netlify-plugin-nextjs",
     "version": "1.0.3"
+  },
+  {
+    "author": "racoonx2p",
+    "description": "Send an email from Gmail after successful/failed Netlify build.",
+    "name": "Gmail Plugin",
```

```suggestion
    "name": "Gmail",
```

We're also in the process of have consistent naming for plugins - https://github.com/netlify/plugins/issues/208 - so I believe we can drop the the `Plugin` here CC @erezrokah